Makefile

```
@@ -1,11 +1,33 @@
 SOURCE_FILES := $(shell find . -type f -name '*.go')
+BIN_DIR := $(abspath bin)
+
+KWCTL_VER := v0.2.5
```

```suggestion
KWCTL_VER := latest
```

So that we don't have to follow up on new releases and/or create some automation to push this dependency here. I think we should respect the `kwctl annotate` contract from this point on, and in general, kwctl usage on the templates.